bedrock/base/management/commands/get_geoip_db.py

```diff
@@ -0,0 +1,25 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+import requests
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        db_url = settings.MAXMIND_DB_URL
+        if not db_url:
+            return
```

Comment:
Maybe we want to add some `logging.debug` here to say that we skipped downloading